core/src/main/java/org/apache/brooklyn/util/core/task/BasicExecutionManager.java

```diff
@@ -507,23 +517,14 @@ public T call() {
             try {
                 T result = null;
                 Throwable error = null;
-                String oldThreadName = Thread.currentThread().getName();
                 try {
-                    if (RENAME_THREADS) {
```

Comment:
I presume this is removed from here because `internalBeforeStart()` and `internalAfterEnd()` are already doing the thread renaming? If so, then great.